WolvenKit.Modkit/RED4/Pack.cs

```diff
@@ -34,7 +34,17 @@ public bool Pack(DirectoryInfo infolder, DirectoryInfo outpath, string? modname
             var outFile = Path.Combine(outpath.FullName, $"{infolder.Name}.archive");
             if (modname != null)
             {
-                outFile = Path.Combine(outpath.FullName, $"{modname}.archive");
+                var sanitizedModName = new string(modname.Where(c => !Path.GetInvalidFileNameChars().Contains(c)).ToArray()).Trim();
```

Comment:
Look into `StringPathExtensions.cs`, this looks like it could go there (or as if you could use methods from there)

Reply:
> I purposely avoid using that because it doesn't exactly work for some of the trails I did. however, moving it to "INVALID NAME" wasn't working either, so I'll fix that too.

You could fix the functions? We're trying to cut down on code duplication regarding string path validation stuff :)

> We discussed this a bit on the discord, but we should try to limit how much we're restricting the user to name their work.
As we have seen, users can't be trusted :P But the fields should act the same IMO